src/gatsby/gatsby-browser.js

```diff
@@ -1,4 +1,5 @@
 import "@bcgov/bootstrap-theme/dist/css/bootstrap-theme.min.css"
+import "@bcgov/bc-sans/css/BC_Sans.css"
```

Comment:
How was the font getting imported previously? Was it coming from the bootstrap theme?

Reply:
Yes, I assume it was from the bootstrap: https://github.com/bcgov/bootstrap-theme/tree/master
However, it hasn't been updated for a while so I imported the font file.